app/vlstorage/main.go

```diff
@@ -401,4 +408,9 @@ func writeStorageMetrics(w io.Writer, strg *logstorage.Storage) {
 	metrics.WriteCounterUint64(w, `vl_rows_dropped_total{reason="too_small_timestamp"}`, ss.RowsDroppedTooSmallTimestamp)
 }
 
+func writeNetstorageInsertMetrics(w io.Writer, strg *netinsert.Storage) {
+	metrics.WriteGaugeUint64(w, `vl_insert_active_streams`, strg.GetActiveStreams())
+	metrics.WriteCounterUint64(w, `vl_insert_send_failed`, strg.SendFailed.Load())
```

Comment:
The name for this metric should conform [Prometheus naming convention](https://docs.victoriametrics.com/victorialogs/quickstart/#building-from-source-code).